src/Passes/Source/Generators/LlvmPassesConfig.hpp

```diff
@@ -0,0 +1,59 @@
+#pragma once
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "Commandline/ConfigurationManager.hpp"
+
+namespace microsoft
+{
+namespace quantum
+{
+
+    class LlvmPassesConfiguration
+    {
+      public:
+        // Default constructor which sets the standard pipeline.
+        LlvmPassesConfiguration();
+
+        // Setup and pre-fabricated configurations
+
+        /// Setup function that registers the different LLVM passes available via LLVM component.
+        void setup(ConfigurationManager& config);
+
+        /// Static function creates a new configuration where all transformations/validation requirements
+        /// are disabled. Using this configuration is equivalent to
+        static LlvmPassesConfiguration disable();
```

Comment:
Something is missing in the end of the comment.

Reply:
Fixed.